.github/CONTRIBUTING.md

```diff
@@ -5,56 +5,85 @@ We love to have your help to make php-webdriver better!
 
-    ./vendor/bin/phpunit
 
-If you want to execute just the unit tests, run:
+For easier development there are also few other prepared commands:
+- `composer fix` - to auto-fix the codestyle and composer.json
+- `composer analyze` - to run only code analysis (without tests)
+- `composer test` - to run all tests
 
-    ./vendor/bin/phpunit --testsuite unit
+### Unit tests
 
-For the functional tests you must first [download](http://selenium-release.storage.googleapis.com/index.html) and start
-the selenium standalone server, start the local PHP server which will serve the test pages and then run the `functional`
-test suite:
+There are two test-suites: one with **unit tests** only (`unit`), and second with **functional tests** (`functional`),
+which requires running Selenium server and local PHP server.
 
-    java -jar selenium-server-standalone-3.9.1.jar -log selenium.log &
-    php -S localhost:8000 -t tests/functional/web/ &
-    ./vendor/bin/phpunit --testsuite functional
+To execute **all tests** in both suites run:
 
-The functional tests will be started in HtmlUnit headless browser by default. If you want to run them in eg. Firefox,
-simply set the `BROWSER_NAME` environment variable:
+```sh
+composer test
+```
 
-    ...
-    export BROWSER_NAME="firefox"
-    ./vendor/bin/phpunit --testsuite functional
+If you want to execute **just the unit tests**, run:
 
-To test with Geckodriver, [download](https://github.com/mozilla/geckodriver/releases) and start the server, then run:
+```sh
+composer test -- --testsuite unit
+```
 
-    export GECKODRIVER=1
-    export BROWSER_NAME=firefox
-    ./vendor/bin/phpunit --testsuite functional
+**Functional tests** are run against a real browser. It means they take a bit longer and also require an additional setup:
+you must first [download](https://www.selenium.dev/downloads/) and start the Selenium standalone server,
+then start the local PHP server which will serve the test pages and then run the `functional` test suite:
 
-### Check coding style
+```sh
+export BROWSER_NAME="htmlunit" # see below for other browsers
+java -jar selenium-server-standalone-X.X.X.jar -log selenium.log &
+php -S localhost:8000 -t tests/functional/web/ &
+# Use following to run both unit and functional tests
+composer all
+# Or this to run only functional tests:
+composer test -- --testsuite functional
+```
 
-Your code-style should comply with [PSR-2](http://www.php-fig.org/psr/psr-2/). To make sure your code matches this requirement run:
+If you want to run tests in different browser then "htmlunit" (Chrome or Firefox), you need to setup the browser driver (Chromedriver/Geckodriver), as it is [explained in wiki](https://github.com/php-webdriver/php-webdriver/wiki/Chrome)
+and then the `BROWSER_NAME` environment variable:
 
-    composer codestyle:check
+```sh
+...
+export BROWSER_NAME="chrome"
+composer all
```

Comment:
We could suggest one line commands ?
```suggestion
BROWSER_NAME="chrome" composer all
```

Reply:
I prefer the export variant, it is more readable for me :shrug: